sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -74,10 +74,12 @@ class BlobServiceClient(StorageAccountHostsMixin):
     :param credential:
```

Comment:
How do we like the rewriting. Open to suggestions and formatting, since I sort of just tacked it on at the end

Reply:
LGTM! Thanks